src/algebra/tropical/basic.lean

```diff
@@ -240,13 +269,11 @@ instance [add_monoid R] : monoid (tropical R) :=
   mul_one := λ _, untrop_injective (add_zero _),
   ..tropical.semigroup }
```

Comment:
Maybe out of scope for this PR, but you should probably add an `npow` field here implemented in terms of `nsmul`, so that `untrop_pow` is just `rfl`.